### ISSUE TITLE
Website: Some changelog entries have very large text

### ISSUE THREAD

**Issue by lydell** [jashkenas/coffeescript#4702]
Looks like the changelog entries for 1.11.0 and 1.10.0 have much larger text than all the other entries.

![screenshot from 2017-09-18 18-44-30](https://user-images.githubusercontent.com/2142817/30553464-9487a49a-9ca1-11e7-82c2-6f0cf004e5c8.png)

**Comment by GeoffreyBooth** [comment]
Look, those were really important updates.

**Comment by jashkenas** [comment]
Actually, @GeoffreyBooth (although I can do it if you like) — would you mind turning down the font-size a bit in general? At least for medium-width displays like laptops? It's currently a little crowded.

I think 13px for the code, and 1.1em for the body text (instead of the current 16 and 1.3) would look great.

**Comment by jashkenas** [comment]
Nevermind, I got it. (Sort of halfway between).

**Comment by lydell** [comment]
I figured out why text gets so big.

```css
.main p, .main li, .main td, .main th {
  font-size: 1.2em;
}
```

The 1.11.0 and 1.10.0 sections seems to be the only ones with `<p>` tags inside the `<li>`s. So the `<li>`s get 20% bigger (as expected), but then the `<p>` tags get 20% bigger on top of that. Then it gets even worse for the nested list.

**Comment by GeoffreyBooth** [comment]
@lydell if you want to fix the child list items, please feel free, or I can do it tonight.

The main body font size I think is the same as Bootstrap’s standard, and I feel like there was a reason to keep it that way, so I would be hesitant to change that. Or at least test thoroughly before pushing a change to that.

Be careful if you touch the styles for the CodeMirror (editable) blocks. There’s a funky trick going on where it’s static HTML that gets replaced with an editable CodeMirror when you mouseover the block, so that we can avoid initializing two dozen CodeMirror editors on load. The styles need to match in both versions, so that the initialization is invisible.

I guess any changes you might be unsure about, or that might be wider in scope, would be best to open a PR for.

**Comment by jashkenas** [comment]
Yeah, maybe let's just set pixel sizes and be done with it. I think I messed up the codemirror swap-ins — sorry!

**Comment by GeoffreyBooth** [comment]
Yeah, @jashkenas can you undo that? The code blocks are popping in size dramatically now.

I’ll look at this soon. I think these changes should get a PR and review.

**Comment by jashkenas** [comment]
I just fixed it in c17b6b8863490c9bbb7dfad140bc600f195c685f instead of reverting. My bad.
